packages/legend-application-studio/src/stores/graphModifier/StoreRelational_GraphModifierHelper.ts

```diff
@@ -421,3 +429,66 @@ export const rootRelationalSetImp_setPropertyMappings = action(
     );
   },
 );
+
+// --------------------------------------------- Post-Processor -------------------------------------
+
+export const relationalDatabaseConnection_addPostProcessor = action(
+  (
+    connectionValueState: RelationalDatabaseConnectionValueState,
+    postProcessor: PostProcessor,
+    observerContext: ObserverContext,
+  ): void => {
+    addUniqueEntry(
+      connectionValueState.connection.postProcessors,
+      observe_PostProcessor(postProcessor, observerContext),
+    );
+  },
+);
+
+export const relationalDatabaseConnection_deletePostProcessor = action(
+  (
+    connectionValueState: RelationalDatabaseConnectionValueState,
+    postProcessor: PostProcessor,
+  ): void => {
+    deleteEntry(connectionValueState.connection.postProcessors, postProcessor);
+  },
+);
+
+export const mapperPostProcessor_addMapper = action(
+  (postProcessor: PostProcessor, mapper: Mapper): void => {
+    addUniqueEntry(
+      (postProcessor as MapperPostProcessor).mappers,
+      observe_Mapper(mapper),
+    );
+  },
+);
+
+export const mapperPostProcessor_mapper_setFrom = action(
```

Comment:
no, you only need to do one level for the target, so the format is `X_Y`,
X = the target, Y = the action

so let's rename these again, for example, here you would have `mapper_setFrom`

Reply:
i see now 🥲